frontend/packages/dev-console/integration-tests/features/vulnerability/project_vulnerability_ODC-4882

```diff
@@ -0,0 +1,102 @@
+   When user goes to project tab on sidebar
+   And user sees Status section in overview tab
+   And user sees vulnerabilities with number mentioned in Status section
+   And user clicks on the Image Vulnerabilities
+   Then user sees Image Vulnerability breakdown pop over
+   And user sees Vulnerable Container Images and Fixable Container Images
+   And user closes the pop over 
+
+@regression
+Scenario: Going to Image Manifest Vulnerabilities page from status in project overview    
+   Given user is in project tab
+   And user sees vulnerabilities with number mentioned in Status section
+   When user clicks on the Image Vulnerabilities
+   And user sees Image Vulnerability breakdown pop over
+   And user sees View all link on bottom of pop over
+   And user clicks on the View all link
+   Then user can see Image Manifest Vulnerabilities page
+
+@regression
+Scenario: Going ImageManifestVuln from status in project overview    
+   Given user is in project tab
+   And user sees vulnerabilities with number mentioned in Status section
+   When user clicks on the Image Vulnerabilities
+   And user sees Image Vulnerability breakdown pop over
+   And user sees vulnerabilities with number mentioned in Status section
+   And user clicks on the image name to open that particular ImageManifestVuln
+   Then user can see ImageManifestVuln for the image name
+
+@regression
+Scenario: Going to quay page from status in project overview    
+   Given user is in project tab
+   And user sees vulnerabilities with number mentioned in Status section
+   When user clicks on the Image Vulnerabilities
+   And user sees Image Vulnerability breakdown pop over
+   And user sees Fixable Container Images section
+   And user sees fixable link for namespaces
+   And user clicks on the fixable link 
+   Then user can see the image open in quay.io
+
+@regression
+Scenario: Vulnerability tab in developer console    
+   Given user is in topology of a project
+   When user goes to project tab on sidebar
+   And user sees vulnerability tab
+   And user clicks on the vulnerability tab
+   Then user sees Image Manifest vulnerabilities
+   And user sees filter with name option selected
+   And user can see image name, namespace, highest severity, pods,fixable and manifests
+   
+@regression
+Scenario: Filter in vulnerability tab
+   Given user is in project tab
+   When user clicks on vulnerability tab
+   And user sees filter with name option selected
+   And user clicks on dropdown associated with filter
+   And user sees label option in the dropdown
+   And user clicks on label option
+   Then user can see filter with label option selected
+
+@regression
+Scenario: Image manifests vulnerability details page
+   Given user is in vulnerability tab
+   And vulnerability tab has vulnerabilities
+   When user clicks on first vulnerability
+   Then Image manifests vulnerability details page opens
+   And ImageManifestsVuln Details page has user details, yaml and affected pods tabs
```

Comment:
```suggestion
   And ImageManifestsVulnerability Details page has image manifest vulnerability details, yaml and affected pods tabs
```

Reply:
`ImageManifestsVuln` name is mentioned on the page that's why I've used it